src/main/webapp/blobstoreFunctions.js

```diff
@@ -0,0 +1,54 @@
+/** Loads photo */
+function loadScripts() {
+  fetchBlobstoreUrl();
+  populatePosts();
+}
+
+
+/** Loads upload image URL and attaches it to the form */
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url') 
+      .then((response) => {
+        return response.text();
+      })
+      .then((imageUploadUrl) => {
+        const messageForm = document.getElementById('plant-pic-form');
+        messageForm.action = imageUploadUrl;
+      });
+}
+
+
+/**
+  * Creates and adds a photo 
+  * pic is the URL of image to be shown in the post
+*/
+function createListElement(pic) {
+  let card = document.createElement("div");
+  card.className = "card m-1";
+
+  let body = document.createElement("div");
+  body.className = "card-body";
+
+  let photo = document.createElement("img");
+  photo.style.width = "100%";
```

Comment:
could you maybe put this line in the css instead?

Reply:
line 33 i mean because I think if we use this line to make more images in the future we will want them to all have this style property

Reply:
should I just take this out for now, since we'll probably be using different size constraints when we add it to the quizzes? I could also make this the page where you can see the picture you currently have, but the whole page will probably need to be restyled. If that's the case I can add just this to the css for now

Reply:
If you don't need it then I would take it out, I wouldn't worry about creating another html page right now

Reply:
sounds good, I took it out